docs/src/pages/customization/typography/typography.md

```diff
@@ -73,7 +74,7 @@ const theme = createMuiTheme({
 <ThemeProvider theme={theme}>
   <CssBaseline />
   {children}
-</ThemeProvider>
+</ThemeProvider>;
```

Comment:
This `;` sounds off

Reply:
It's from prettier. If the semicolon is "off" then so is the code.